runtime/tests/CMakeLists.txt

```diff
@@ -58,3 +58,4 @@ add_subdirectory(vm_lifecycle)
 if(OMR_OS_ZOS)
 	add_subdirectory(zos)
 endif()
+add_subdirectory(sharedclassapi)
```

Comment:
Please keep the subdirectories sorted alphabetically; add this line after line 48:
```
add_subdirectory(shared)
add_subdirectory(sharedclassapi)
```